tutorials/service-accounts/subscription-echo-read.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START config_connector_service_accounts_subscription]
+apiVersion: pubsub.cnrm.cloud.google.com/v1alpha2
+kind: PubSubSubscription
+metadata:
+  name: echo-read
+spec:
+  ackDeadlineSeconds: 15
```

Comment:
The gcloud command at https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform appears to not set as many options:

```gcloud pubsub subscriptions create echo-read --topic=echo```

It seems like the two should match?